app/app/calculation_service/model/linear_model.py

```diff
@@ -112,10 +113,14 @@ def get_rep_n_from_study_design(self, study_design):
         return study_design.isu_factors.smallest_group_size
 
     def calculate_c_matrix(self, predictors):
-        partials = [self.calculate_partial_c_matrix(p) for p in predictors if p.in_hypothesis]
-        partials.append(np.matrix(np.identity(1)))
-        c_matrix = kronecker_list(partials)
-        return c_matrix
+        try:
```

Comment:
Is an exception ever thrown here?
An exception doesn't seem appropriate for this.
We should instead be verifying our data model and sending back errors if there are problems.
remove this try/catch for now.